app/scenes/Document/components/EditableTitle.tsx

```diff
@@ -195,13 +278,32 @@ const Title = styled(ContentEditable)<TitleProps>`
         opacity: 1;
       }
     }
+
+    ${AnimatedEmoji} {
```

Comment:
If you hover over the title and move the mouse slowly left there is a few pixels where the emoji button disappears and no longer counts as hovering.